elasticsearch/run.sh

```diff
@@ -85,7 +85,7 @@ wait_for_port_open() {
 do
     echo -n "."
     sleep $RETRY_INTERVAL
-    (( retry -= 1 ))
+    (( retry -= 1 )) || :
```

Comment:
Amazing to see emoticons had a real meaning before they [become](https://www.w3.org/2005/Incubator/emotion/) a thing.
Thanks Rich.